components/bsx/Offer/OfferTable.vue

```diff
@@ -160,29 +159,35 @@ export default class OfferTable extends mixins(OfferMixin) {
     return this.$store.getters.currentUrlPrefix
   }
 
-  get total(): number {
-    return this.offers.length
+  @Watch('currentPage')
+  watchPageValue(val) {
+    this.replaceUrl(String(val))
   }
 
-  get showList(): any[] {
-    const endIndex = this.currentPage * this.itemsPerPage
-    return this.displayOffers(this.offers).slice(
-      endIndex - this.itemsPerPage,
-      endIndex
-    )
+  @Debounce(100)
```

Comment:
I remember asking you why you need to use a debounce, can't find the answer anymore

Reply:
To avoid Modifying the url frequently.
If `page` was changed by many places in a short time, we just need to modify the router once.